src/gui/advancedsettings.h

```diff
@@ -51,6 +51,7 @@ public slots:
 
 private slots:
     void updateCacheSpinSuffix(int value);
+    void updateSaveResumeDataIntervalSuffix(int value);
```

Comment:
Implementation has `const int value`

Reply:
`const` is meaningless and confusing in declaration (unless it's pointer/reference to const). We discussed it earlier.

Reply:
And Yes, we can have it this way, with `const` only in definition.

Reply:
AFAIK that it doesn't make sense to have `const` for value types variable in declaration signature.
IIRC glassez mentioned elsewhere about this.